cmake/packaging/rpm.cmake

```diff
@@ -176,6 +176,15 @@ macro(ov_cpack_settings)
         set(gpu_copyright "generic")
     endif()
 
+    # openvino-nvidia-gpu
+    if(ENABLE_NVIDIA)
+        set(CPACK_COMPONENT_NVIDIA_DESCRIPTION "OpenVINO™ NVIDIA GPU plugin is developed in order to enable deep neural networks inference on NVIDIA GPUs, using OpenVINO™ API")
+        set(CPACK_RPM_NVIDIA_PACKAGE_REQUIRES "${core_package}")
+        set(CPACK_RPM_NVIDIA_PACKAGE_NAME "openvino-gpu-plugin-${cpack_name_ver}")
+        
+        set(nvidia_copyright "generic")
```

Comment:
I'm not sure that current "generic" copyright contains information about CUDA. 
Looks like new copyright file should be created for NVIDIA plugin from this content https://github.com/openvinotoolkit/openvino_contrib/blob/master/third-party-programs.txt#L2893-L3006